cxx-standard.cmake

```diff
@@ -64,7 +64,10 @@ macro(CHECK_MINIMAL_CXX_STANDARD STANDARD)
     if(_COMPILER_DEFAULT_CXX_STANDARD EQUAL 199711
         OR (_COMPILER_DEFAULT_CXX_STANDARD EQUAL 201103 AND _MINIMAL_CXX_STANDARD GREATER 11)
         OR (_COMPILER_DEFAULT_CXX_STANDARD EQUAL 201402 AND _MINIMAL_CXX_STANDARD GREATER 14)
-        OR (_COMPILER_DEFAULT_CXX_STANDARD EQUAL 201703 AND _MINIMAL_CXX_STANDARD GREATER 17))
+        OR (_COMPILER_DEFAULT_CXX_STANDARD EQUAL 201703 AND _MINIMAL_CXX_STANDARD GREATER 17)
```

Comment:
While we are touching this file we could update it for C++20 since the standard has been released

Reply:
Done - please verify this is as intended